website/docs/limitations.md

```diff
@@ -0,0 +1,28 @@
+---
+id: limitations
+title: Limitations
+description: A limitations table where every technical and business limitation is collected for each subscription plan.
+---
+
+## A limitations table where every technical and business limitation is collected for each subscription plan.
+
+:::info Every bolded value in the table marks a technical limitation. Meaning that the value can be increased by [contacting us](https://configcat.com/support/).
+:::
+
+|                                                          | Free  | Pro   | Smart    | Enterprise | Dedicated | Student | Teacher |
+| -------------------------------------------------------- | ----- | ----- | -------- | ---------- | --------- | ------- | ------- |
+| Config JSON downloads  (million)                         | 5     | 25    | 250      | 1000       | 5000      | 25      | 5       |
+| Network traffic (GB/month)                               | 20    | 100   | 1024     | 4096       | 24576     | 100     | 100     |
+| Products                                                 | 1     | 3     | **1000** | **1000**   | **1000**  | 3       | 1000    |
+| Environments (per product)                               | 2     | 3     | **50**   | **50**     | **50**    | 3       | 2       |
+| Segments (per product)                                   | 2     | 3     | **500**  | **1000**   | **1000**  | 3       | 3       |
+| Feature flags / Settings                                 | 10    | 100   | **500**  | **500**    | **500**   | 1000    | 10      |
+| % options (per feature flag)                             | 4     | 8     | **500**  | **500**    | **500**   | 8       | 4       |
+| Targeting rules (per feature flag)                       | 4     | 8     | **500**  | **500**    | **500**   | 8       | 4       |
```

Comment:
I think we should add the "AND" condition technical limit after this line. 
So on the new UI you'll be able to define targeting rules with AND conditions. The technical limit for the AND condition count is 100.